onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -419,10 +429,12 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             # the input_hidden_size represents the input hidden size, this is used as needed but hidden sizes for Q, K are extracted appropriately
             new_node = self.create_attention_node(mask_index, matmul_q, matmul_k, matmul_v, add_q, add_k, add_v,
                                                   q_num_heads, self.hidden_size, root_input,
-                                                  attention_last_node.output[0])
+                                                  attention_last_node.output[0], add_qk_str)
             if new_node is None:
                 return
 
+            print("New node created")
```

Comment:
Looks like debug information, how about logger.debug(...)?

Reply:
this was a local debug, removed.